flixel/addons/editors/tiled/TiledLayer.hx

```diff
@@ -18,6 +18,8 @@ class TiledLayer
 	public var opacity:Float;
 	public var visible:Bool;
 	public var properties:TiledPropertySet;
+	public var offsetx:Float;
```

Comment:
Another option might be using a `FlxPoint` object, that'd make it consistent with `FlxSprite`'s `offset`...